lib/packetgen/header.rb

```diff
@@ -89,8 +90,16 @@ def self.get_header_class_by_name(name)
 require_relative 'header/arp'
 require_relative 'header/ipv6'
 require_relative 'header/icmpv6'
+require_relative 'header/gre'
 require_relative 'header/udp'
 require_relative 'header/tcp'
+require_relative 'header/eap'
 require_relative 'header/esp'
+require_relative 'header/ike'
 require_relative 'header/dns'
+<<<<<<< HEAD
 require_relative 'header/http'
+=======
+require_relative 'header/asn1_base'
+require_relative 'header/snmp'
+>>>>>>> sdaubert/master
```

Comment:
You had not resolved conflicts before committing.

You should revert to previous commit, redo the marge, resolve conflicts and then push (with `--force`):
```
### maybe do a git merge --abort
git reset --hard 75474fb5
git pull https://github.com/sdaubert/packetgen.git master
## Here edit you conflicted files an resolve conflicts
git commit
git push --force
```

Reply:
Oh, the joys of `git` 😹 

I'll look into trying to fix this ASAP. Or I'll just make another, clean PR to fix the issues if I can't fix this. Sorry for the inconvenience!